config/settings.defaults.js

```diff
@@ -171,7 +171,7 @@ module.exports = {
   maxJsonRequestSize:
     parseInt(process.env.MAX_JSON_REQUEST_SIZE, 10) || 8 * 1024 * 1024,
 
-  dispatcherCount: process.env.DISPATCHER_COUNT,
+  dispatcherCount: process.env.DISPATCHER_COUNT || 10,
```

Comment:
I'm not sure how often we'd want to change this, but as it stands the setting would come through as a string.
```suggestion
  dispatcherCount: parseInt(process.env.DISPATCHER_COUNT || 10, 10)
```

Reply:
thanks, we actually never set the DISPATCHER_COUNT in prod, instead always relying on the default. Will change.